tests/integration/states/test_tag.py

```diff
@@ -25,16 +28,39 @@ def test_manage_update(patch_salt_globals_tag_state):
     """
     test tag manage update
     """
-    res = tagging_state.present("state tag", description="new discription")
-    assert "updated" in res["comment"]
+    tagging_state.present_category("state test cat", ["string"], "SINGLE", "test category")
+    cats = tagging.list_category()
```

Comment:
**question** doesn't this actually create a category? Also, maybe we should make this a fixture so it can create/remove the category as well.